source/plugin/Assets/GoogleMobileAds/Common/DummyClient.cs

```diff
@@ -128,31 +151,70 @@ public int GetDeviceSafeWidth()
         public void CreateBannerView(string adUnitId, AdSize adSize, AdPosition position)
         {
             Debug.Log("Dummy " + MethodBase.GetCurrentMethod().Name);
+            if (inUnityEditor) {
+                dummyAd = Resources.Load(prefabAds[adSize]) as GameObject;
```

Comment:
What happens if this comes back null?

Reply:
I have null checks in the DummyAdBehaviour methods so that nothing will actually happen if they are called except for a log